README.md

```diff
@@ -16,6 +16,10 @@
 ### 1. Install plugin
 
 ```shell
+  yarn add -D eslint-plugin-ember
```

Comment:
This is totally good, but I generally prefer (especially in documentation or shared shell scripts) to use the longer form (because its much clearer generally speaking than a single cryptic character).

In this case that would be:

```sh
yarn add --dev eslint-plugin-ember
```

What do you think?

Reply:
Makes sense :) I'll fix this.